configs/platforms/debian-6-i386.rb

```diff
@@ -0,0 +1,11 @@
+platform "debian-6-i386" do |plat|
```

Comment:
I think Razor would be noarch, so we should drop 50% of these platform files (if I am correct about norachness)